src/main/java/com/puppycrawl/tools/checkstyle/checks/AbstractTypeAwareCheck.java

```diff
@@ -209,6 +209,7 @@ protected static boolean isSubclass(Class<?> child, Class<?> parent) {
     }
 
     /**
+     * If classresolver is not initialised then it will be initialised and returned.
```

Comment:
First sentence should say what the method does, not any of it's internal workings.
Please change it to something like `Returns the current tree's ClassResolver.` or something else that doesn't imitate the `return` description so much.